Source/RunActivity/Viewer3D/Popups/HUDWindow.cs

```diff
@@ -1009,8 +1009,7 @@ void TextPageForceInfo(TableData table)
 
                 if (train.TrainWindResistanceDependent) // Only show this information if wind resistance is selected
                 {
-                    TableAddLine(table, "Wind Speed: {0:N2} mph   Wind Direction: {1:N2} Deg   Train Direction: {2:N2} Deg    ResWind: {3:N2} Deg   ResSpeed {4:N2} mph", Me.ToMi(pS.TopH(train.PhysicsWindSpeedMpS)), train.PhysicsWindDirectionDeg, train.PhysicsTrainLocoDirectionDeg, train.ResultantWindComponentDeg, Me.ToMi(pS.TopH(train.WindResultantSpeedMpS)));
-
+                    TableAddLine(table, $"{Viewer.Catalog.GetString("Wind Speed:")} {Me.ToMi(pS.TopH(train.PhysicsWindSpeedMpS)):N2} mph   {Viewer.Catalog.GetString("Wind Direction:")} {train.PhysicsWindDirectionDeg:N2} Deg   {Viewer.Catalog.GetString("Train Direction:")} {train.PhysicsTrainLocoDirectionDeg:N2} Deg    {Viewer.Catalog.GetString("ResWind:")} {train.ResultantWindComponentDeg:N2} Deg   {Viewer.Catalog.GetString("ResSpeed:")} {Me.ToMi(pS.TopH(train.WindResultantSpeedMpS)):N2} mph");
```

Comment:
You can also put the old string with the parameters inside within a GetStringFmt function.
The new string seems really complicated and doesn't translate the units.

Reply:
Formatted the new string. Now it looks easier to read and added the GetStringFmt () function.

Reply:
Maybe [FormatStrings.FormatSpeed()](https://github.com/openrails/openrails/blob/4a5757f158356a6cd9444b355d9e51d281e41fa3/Source/ORTS.Common/Conversions.cs#L580) is what we want here?

Reply:
This error is simpler and is related to the strings: "Wind speed:", "Wind direction:", "Train direction:", "ResWind:", "ResSpeed:" that they do not appear in the *. pot for translation, because the Viewer.Catalog.GetString () function was missing.
Here is the proposed code: https://github.com/openrails/openrails/pull/464/commits/1d09b79d180b02903267fddf3462e7d1cf80628c

Is it mandatory to translate the units "Deg" and "mph"?  I have not found information about it.

Reply:
Hi Mauricio,

I can find no previous translations from "Deg" or "mph" either, so I guess we can leave that alone.

Chris